MatchHandler.js

```diff
@@ -0,0 +1,16 @@
+class MatchHandler
```

Comment:
zazwyczaj klaska która jest 1:1 proxy na obiekt pod maską (tutaj dwie metody, każda metoda to jedna linijka - zwykłe wywołanie metody z obiektu którym zarządza) jest zawsze przerostem formy nad treścią. Wyjątek od tego stanowi adapter miedzy jakimiś frameworkami, warstawami albo innymi poziomami abstrakcji.

Reply:
Chyba że jest to początek czegos większego, ale nie widzem na razie ;) A zgodnie z brzytwą ockhama niepotrzebne byty należy eliminować :)

Reply:
Początek czegoś większego :D